detox/e2e/support/server_api/plugin.js

```diff
@@ -18,41 +18,84 @@ import {apiUploadFile, getResponseFromError} from './common';
 // - return value defined by `@return`
 // ****************************************************************
 
+const prepackagedPlugins = [
+    'antivirus',
+    'mattermost-autolink',
+    'com.mattermost.aws-sns',
+    'com.mattermost.plugin-channel-export',
+    'com.mattermost.custom-attributes',
+    'github',
+    'com.github.manland.mattermost-plugin-gitlab',
+    'com.mattermost.plugin-incident-management',
+    'jenkins',
+    'jira',
+    'com.mattermost.nps',
+    'com.mattermost.welcomebot',
+    'zoom',
+];
+
 /**
- * Get plugins.
- * See https://api.mattermost.com/#tag/plugins/paths/~1plugins/get
- * @return {Object} returns {plugins} on success or {error, status} on error
+ * Disable non-prepackaged plugins.
  */
-export const apiGetAllPlugins = async () => {
+export const apiDisableNonPrepackagedPlugins = async () => {
```

Comment:
I don't see this being used right now but it should be called at `init.js` / `beforeAll`.

Reply:
Added and tested